src/link.js

```diff
@@ -224,23 +223,19 @@ export default class Link extends Plugin {
 	 */
 	_attachPanelToElement( parentLink ) {
 		const viewDocument = this.editor.editing.view;
-		const domEditableElement = viewDocument.domConverter.getCorrespondingDomElement( viewDocument.selection.editableElement );
 		const targetLink = parentLink || getPositionParentLink( viewDocument.selection.getFirstPosition() );
 
-		// When selection is inside link element, then attach panel to this element.
-		if ( targetLink ) {
-			this.balloonPanelView.attachTo(
-				viewDocument.domConverter.getCorrespondingDomElement( targetLink ),
-				domEditableElement
-			);
-		}
-		// Otherwise attach panel to the selection.
-		else {
-			this.balloonPanelView.attachTo(
-				viewDocument.domConverter.viewRangeToDom( viewDocument.selection.getFirstRange() ),
-				domEditableElement
-			);
-		}
+		const target = targetLink ?
+				// When selection is inside link element, then attach panel to this element.
+				viewDocument.domConverter.getCorrespondingDomElement( targetLink )
+			:
+				// Otherwise attach panel to the selection.
+				viewDocument.domConverter.viewRangeToDom( viewDocument.selection.getFirstRange() );
+
+		this.balloonPanelView.attachTo( {
+			target,
+			limiter: viewDocument.domConverter.getCorrespondingDomElement( viewDocument.selection.editableElement )
```

Comment:
Yes. If the balloon does not fit into the limiter, the algorithm considers

1. the order of `#positions` and prefers the first one,
2. fitting into the viewport, so the algorithm will chose such position that is still visible to the end–user.

So if the nested editable is visible in the viewport, it basically will be the very first position, which is 

```
 * * South east:
 *
 *		[ Target ]
 *		    ^
 *		+-----------------+
 *		|     Balloon     |
 *		+-----------------+
```
https://github.com/ckeditor/ckeditor5-ui-default/blob/t/131/src/balloonpanel/balloonpanelview.js#L167-L173

We may later change it to SW in RTL environments, but it's not a big deal ATM.